lib/ansible/modules/files/fetch.py

```diff
@@ -20,6 +20,7 @@
 description:
 - This module works like M(copy), but in reverse.
 - It is used for fetching files from remote machines and storing them locally in a file tree, organized by hostname.
+- Files that already exist at C(dest) will be overwritten if they are different than the C(src).
```

Comment:
```suggestion
- Files that already exist at I(dest) will be overwritten if they are different than the I(src).
```
`I(...)` is used for option names, `C(...)` for option values.

Reply:
Understood. Thanks.